renderer/views/player.js

```diff
@@ -43,7 +43,7 @@ function renderMedia (state) {
       mediaElement.play()
     }
     // When the user clicks or drags on the progress bar, jump to that position
-    if (state.playing.jumpToTime) {
+    if (state.playing.jumpToTime != null) {
```

Comment:
I found it on [Stack Overflow](http://stackoverflow.com/a/15992131/4132790) and liked it better than `jumpToTime === null || jumpToTime === undefined` or anything close to that. Brief investigation confirmed this should work as desired.